im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,425 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
```

Comment:
Assuming this journal only publishes in German, the style should be fixed to this language, which can be done by adding `default-locale="de-DE"` to the `<style>` tag.